homeassistant/components/template/light.py

```diff
@@ -48,7 +47,7 @@
 )
 
 _LOGGER = logging.getLogger(__name__)
-_VALID_STATES = [STATE_ON, STATE_OFF, "true", "false"]
+_VALID_STATES = [LightState.ON, LightState.OFF, "true", "false"]
```

Comment:
These are valid strings coming from the rendered template, I believe. Then we should not use the state enum here.